ingest/src/test/java/com/microsoft/azure/kusto/ingest/IngestClientImplTest.java

```diff
@@ -19,16 +19,13 @@
 class IngestClientImplTest {
```

Comment:
I am missing tests here, including tests to validate all the thrown exceptions.

Reply:
We have another branch for tests, and I have already added these kinds of tests.